src/Xenial.Tasty/Remote/NamedPipeRemoteHook.cs

```diff
@@ -57,8 +58,8 @@ static SerializableTestCase MapToSerializableTestCase(Metadata.TestCase test)
 
             scope.RegisterReporter(test => remote.Report(MapToSerializableTestCase(test)));
 
-            Task SummaryReporter(IEnumerable<Metadata.TestCase> @cases)
-                => remote.Report(@cases.Select(test => MapToSerializableTestCase(test)));
+            Task SummaryReporter(TestSummary summary)
+                => remote.Report(summary.Select(test => MapToSerializableTestCase(test)));
```

Comment:
I guess we change the signature of remote summary reporters in the future?

Reply:
?!?

Reply:
In this form remote reporters still do their stuff on their own and won't get any profit for now, cause it's `IEnumerable<SerializableTestCase>` over the wire. So we can change that to an `SerializableTestSummary` in the future.